strangerstudios/sitewide-sales
# Countdown timer refactor

### All Submissions:






### Changes proposed in this Pull Request:

Refactored blocks by restructuring code files. Incorporated new countdown timer block through modifying 'includes/blocks.php' and 'webpack.config'. Introduced a separate folder in 'sources' for new block files. Transferred code from 'blocks/block.js' and 'blocks/block.php' to individual files within the new block folder.



Resolves XXX.

### How to test the changes in this Pull Request:

1. Make sure there is a sitewide sale active
2. insert countdown timer block in the page editor 
3. View the block on the frontend




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

- Modifcations to blocks.php and webpack.config.js
- New src/countdown-timer folder with block files
